srcpkgs/66/template

```diff
@@ -0,0 +1,18 @@
+# Template file for '66'
+pkgname=66
+version=0.1.0.0
+revision=1
+wrksrc="66-v${version}"
+build_style=gnu-configure
+makedepends="oblibs oblibs-devel skalibs-devel execline-devel s6-devel s6-rc s6-portable-utils"
+depends="s6-rc s6-portable-utils"
+short_desc="Helpers tools around S6 supervision suite"
```

Comment:
This short description sounds wrong. Use `s6-rc`, not `S6`.

Reply:
Indeed it does. It is also the short description in the README file of the project.

Reply:
Pushed proposed change.